malasakit-django/pcari/templates/dev.html

```diff
@@ -0,0 +1,31 @@
+{% extends 'base.html' %}
```

Comment:
Not sure if we want to make this an actual page in our app (may confuse later developers, and we don't really need any Django features). Ideally, this would just be a static page that Apache directly routes to.